File: api-examples/cat/count/python/attribute_search_spatial_circle_responseFilter.py
import requests

url = "https://pudx.catalogue.iudx.org.in/catalogue/v1/count?attribute-name=(tags)&attribute-value=((pollution))&lat=18.4&lon=73.9&radius=200000&attribute-filter=(tags,id)"

payload = {}
headers= {}

response = requests.request("GET", url, headers=headers, data = payload)

print(response.text.encode('utf8'))

File: api-examples/cat/search/Go/attribute_search_spatial_circle_responseFilter.go
package main

import (
  "fmt"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://<catalogue-server-ip>/catalogue/v1/search?attribute-name=(tags)&attribute-value=((pollution))&lat=12.273737&lon=78.37475&radius=200000&attribute-filter=(tags,id)"
  method := "GET"

  client := &http.Client {
  }
  req, err := http.NewRequest(method, url, nil)

  if err != nil {
    fmt.Println(err)
  }
  res, err := client.Do(req)
  defer res.Body.Close()
  body, err := ioutil.ReadAll(res.Body)

  fmt.Println(string(body))
}
